### ISSUE TITLE
Sync sites panel not including all sites

### ISSUE THREAD

**Issue by Gwalior-84** [openmsupply/msupply-dashboard#124]
Dashboard:  https://png.msupply.org:3000/d/KK-crfJWk/sync-statistics-2020?orgId=1

![image](https://user-images.githubusercontent.com/9062300/87391544-493f6080-c5ee-11ea-8bea-1383e08d9ed4.png)

Sync sites panel is only showing 22 sites.  There are approx. 100 in the system.

I expect all 100 to be included.

**Comment by mark-prins** [comment]
There's a filter at the top for sites which is limiting the number shown. That list removes sites which are

- disabled
- have a sync ID of 0 ( or not defined )
- the name is like `Lab`
- the name is like `Dispensary`

removing that restriction shows 122 sites.

**Comment by mark-prins** [comment]
The problem was that the older dashboard report is using a 'sync site name' which does not match store name.
However the query was expecting the two to match. I've corrected for now by using an intermediary table - there are too many problems in matching the names directly (`HC` = `Health Centre` etc). Will create an mSupply issue to create the additional linking required.

**Comment by Gwalior-84** [comment]
Thanks @mark-prins .

How can a store have a sync ID of 0?

**Comment by mark-prins** [comment]
While it's possible, it's not exactly a valid configuration! More likely that is referring to sites which aren't configured for sync. The actual query was for `sync_id > 0` which is basically "is sync configured?"

**Comment by Gwalior-84** [comment]
This panel is still missing sites.  For example, Badili AMS should be included here:

![image](https://user-images.githubusercontent.com/9062300/88604762-7b0af980-d0bb-11ea-8b27-db64b4a343c5.png)

Lae AMS is also missing.

**Comment by Gwalior-84** [comment]
There are other issues with stores not appearing:

![image](https://user-images.githubusercontent.com/9062300/88990565-a131d300-d321-11ea-9fd9-1688b767b046.png)

1. Vanimo store should be in the **Sync stats** table as well as the **Store performance indicators** table
2. Haikost BMU should be on both tables (that just got categorised as a BMU today)

**Comment by mark-prins** [comment]
>Vanimo store should be in the Sync stats table as well as the Store performance indicators table

Only the `Store performance indicators` table had been changed  expand the query. The `Sync stats` one was still just reporting what the old dashboard report shows. I've modified that query to show all stores - some do not have a version showing, I expect that to improve with v4.12.

> Haikost BMU should be on both tables (that just got categorised as a BMU today)

The changes hadn't exported yet. I've manually exported all of the required tables and it shows now.
